scripts/update-charts.sh

```diff
@@ -8,8 +8,13 @@ echo
 
 "
 
+CHARTS=(/inspr-stack /inspr-stack/subcharts/insprd /inspr-stack/subcharts/uidp)
+
 echo "Creating new .tgz file..."
-helm package ../build/helm -d ../build/charts
+for chart in ${CHARTS[@]}; do
+    helm dependency update ../build$chart
+    helm package ../build$chart -d ../build/charts
```

Comment:
Shouldn't it be `../build${chart}` ?  
Also, this script is going to be run manually after the release action passes right?